src/ir/comp.rs

```diff
@@ -328,6 +353,169 @@ impl Ctx<Invoke> for Component {
     }
 }
 
+impl Component {
+    pub fn resolve_prop(&mut self, prop: Prop) -> PropIdx {
```

Comment:
Let's implement this as a Foldable for this logic in a PR after this is merged. Can we open an issue to track that?

Reply:
#246